ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -552,6 +648,31 @@ private boolean isTableSatifiesConfig(Table table) {
     return true;
   }
 
+  private void fetchFailoverMetadata(Hive hiveDb) throws HiveException, TException {
+    FailoverMetaData fmd = new FailoverMetaData(
+            new Path(work.getCurrentDumpPath(), ReplUtils.REPL_HIVE_BASE_DIR), conf);
+    List<Long> txnsForDb = getOpenTxns(getTxnMgr().getValidTxns(excludedTxns), work.dbNameOrPattern);
+    if (!txnsForDb.isEmpty()) {
+      hiveDb.abortTransactions(txnsForDb);
```

Comment:
At least in debug mode a log statement is required

Reply:
Done